messages/commonErrors.md

```diff
@@ -0,0 +1,19 @@
+# error.NoTestsSpecified
+
+You must specify tests using the --tests flag if the --test-level flag is set to RunSpecifiedTests.
+
+# error.InvalidRunTests
+
+runTests can only be used with a testLevel of RunSpecifiedTests.
```

Comment:
```suggestion
runTests can be used only with a testLevel of RunSpecifiedTests.
```

Reply:
@jshackell-sfdc - Esteban mentioned that he got the verbiage for this error from `sf deploy metadata` - I told him not to change it unless you feel like we should change it all places that use this error message. It's a minor geeky grammatical edit.

Reply:
Fixed. Thank you both.